mkosi/__init__.py

```diff
@@ -1331,6 +1376,17 @@ def mount_image(
         yield
     finally:
         with complete_step("Unmounting image"):
+            if realroot is not None:
+                umount(root)
```

Comment:
you unmount root twice now... the next invocation of this is just a few lines below

Reply:
Yes, but I also mount it twice: the first mount is the overlayfs mount, and then I do a bind mount to adjust the location one level up…

Reply:
ah, makes sense. Can you add a comment though?